Assets/MixedRealityToolkit/Utilities/Scripts/Editor/CustomHeaderDrawer.cs

```diff
@@ -1,15 +1,12 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Reflection;
-using System.Text.RegularExpressions;
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
 using UnityEngine;
-#if UNITY_EDITOR
+using MixedRealityToolkit.UtiltiesInspectors;
 using UnityEditor;
-#endif
 
-namespace HoloToolkit.Unity
+namespace MixedRealityToolkit.Utilties
```

Comment:
typo: Utilties -> Utilities (this one is in almost all of them, but I'll just put it once)

Reply:
Resolved